packages/app/src/components/layouts/AuthLayout.tsx

```diff
@@ -137,14 +138,16 @@ export const FOOTER_LINKS: FooterLink[] = [
 ]
 
 export const SEARCH_ITEMS: SearchItems[] = [
-  ...NAV_LINKS,
-  ...FOOTER_LINKS,
   {
     icon: <IconSearch />,
     label: 'profileView.title',
     to: '/profile',
     description: 'profileView.description',
+    color: 'blue',
```

Comment:
Is blue correct? Or should it be the primary variable?

Reply:
The `color` prop is false, it does nothing. I removed it. I also realised, that a hard-coded blue was used in line `203`. I removed that as well, the default is yellow (the matching text is highlighted with the user's search string).